opm/grid/cpgpreprocess/preprocess.c

```diff
@@ -80,6 +81,13 @@ linearindex(const int dims[3], int i, int j, int k)
     return i + dims[0]*(j + dims[1]*k);
 }
 
+static int
+vertical_cart_neighbors(const int dims[3], int i, int j){
+    int k1, k2;
+    k1 = i/dims[0]/dims[1];
+    k2 = j/dims[0]/dims[1];
+    return (k1 - k2) == 1 || (k2 - k1) == -1;
```

Comment:
What is the difference between these two conditions?  Did you mean
```C
(k1 - k2) == 1 || (k2 - k1) == 1
```
instead?

Reply:
Thanks. I blame my underdeveloped multitasking skills. Currently not triggered but will change.